account_payment_order/report/account_payment_order.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="print_account_payment_order_document">
+        <t
+            t-set="doc"
+            t-value="doc.with_context({'lang': doc.generated_user_id and doc.generated_user_id.lang or user.lang})"
+        />
+        <t t-call="web.external_layout">
+            <div class="page">
+                <div class="oe_structure" />
+                <div class="row">
+                    <div class="col-4 offset-8">
+                        <span
+                            t-raw="'%s &lt;br&gt;' % doc.journal_id.bank_id.name if doc.journal_id.bank_id.name else ''"
+                        />
```

Comment:
Using `t-raw` here doesn't seem like a good idea here.
IMO, if we want each field in a new line, we should use `div`.
Using `t-field` will automatically hide it if the field is empty, so it's also nicer

```suggestion
                        <div t-field="doc.journal_id.bank_id.name" />
```

Reply:
This is also resolved